docs/specs/IDE-execution.md

```diff
@@ -71,13 +78,7 @@ The payload is best explained using an example:
     "args": [
```

Comment:
Should these args now apply to all projects from the `launch_configurations` list (same with the `env` property)?

Reply:
> In Aspire version 1 release only a single launch configuration instance, of type `project`, can be used as part of a run session request

Oh, I see that there will be only one project per request in this list.

Reply:
Yes, the idea is that `env` and `args` apply to all configurations that are part of the request and form a "baseline" of sorts. If that is not desired for certain configuration(s), these properties could be overridden on the individual configuration level. We do not have this scenario today, but I could imagine a need like that emerging in future.